src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/DBCaseRepository.java

```diff
@@ -224,9 +224,12 @@ public void saveCaseEvent(String authorisation, Claim claim, CaseEvent caseEvent
     }
 
     @Override
-    public void saveClaimDocuments(String authorisation,
-                                   Long claimId,
-                                   ClaimDocumentCollection claimDocumentCollection) {
+    public Claim saveClaimDocuments(String authorisation,
+                                    Long claimId,
+                                    ClaimDocumentCollection claimDocumentCollection) {
         claimRepository.saveClaimDocuments(claimId, jsonMapper.toJson(claimDocumentCollection));
+        return claimRepository.getById(claimId).orElseThrow(() ->
+            new NotFoundException(
+                String.format("Claim not found by primary key %s. This should never happen", claimId)));
```

Comment:
not keen on including..."this should never happen" in log msg

Reply:
ok will remove that ..